src/main/java/com/cburch/logisim/vhdl/sim/VhdlSimulatorTclBinder.java

```diff
@@ -148,15 +145,13 @@ public void start() {
 
                     vsim.tclStartCallback();
                     return;
-
                   } else if (line.contains(msgCheckOnError)) {
                     // Last line of error log. Do not read next line or it will be stuck
                     break;
                   }
                 }
 
-                MessageBox userInfoBox =
-                    new MessageBox(
+                final var userInfoBox =                    new MessageBox(
```

Comment:
Why more than one space between the = and new?

Shouldn't the next 3 lines be indented 4 spaces, not 8?

Reply:
@davidhutchens  It should. In general formatting of this PR sucks as it appears my formatter silently crashed on these files and I missed to notice the oddities. I will update the PR with proper formatting once I fix the crashes.

Reply:
Fixed.